src/olympia/activity/migrations/0020_auto_20221214_1331.py

```diff
@@ -0,0 +1,27 @@
+# Generated by Django 3.2.15 on 2022-12-14 13:31
+
+from django.db import migrations, models
+import olympia.amo.models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('activity', '0019_merge_0018_auto_20221122_1313_0018_auto_20221128_1706'),
+    ]
+
+    operations = [
+        migrations.RemoveIndex(
+            model_name='iplog',
+            name='log_activity_ip_ip_address_ba36172a',
+        ),
+        migrations.AlterField(
+            model_name='activitylog',
+            name='action',
+            field=models.SmallIntegerField(choices=[(1, 1), (2, 2), (3, 3), (4, 4), (5, 5), (6, 6), (7, 7), (8, 8), (9, 9), (12, 12), (16, 16), (17, 17), (18, 18), (19, 19), (20, 20), (21, 21), (22, 22), (23, 23), (24, 24), (25, 25), (26, 26), (27, 27), (28, 28), (29, 29), (31, 31), (32, 32), (33, 33), (34, 34), (35, 35), (36, 36), (37, 37), (38, 38), (39, 39), (40, 40), (41, 41), (42, 42), (43, 43), (44, 44), (45, 45), (46, 46), (47, 47), (48, 48), (49, 49), (53, 53), (60, 60), (61, 61), (62, 62), (98, 98), (99, 99), (100, 100), (101, 101), (102, 102), (103, 103), (104, 104), (105, 105), (106, 106), (107, 107), (108, 108), (109, 109), (110, 110), (120, 120), (121, 121), (128, 128), (130, 130), (131, 131), (132, 132), (133, 133), (134, 134), (135, 135), (136, 136), (137, 137), (138, 138), (139, 139), (140, 140), (141, 141), (142, 142), (143, 143), (144, 144), (145, 145), (146, 146), (147, 147), (148, 148), (149, 149), (150, 150), (151, 151), (152, 152), (153, 153), (154, 154), (155, 155), (156, 156), (157, 157), (158, 158), (159, 159), (160, 160), (161, 161), (162, 162), (163, 163), (164, 164), (165, 165), (166, 166), (167, 167), (168, 168), (169, 169), (170, 170), (171, 171), (172, 172)]),
+        ),
+        migrations.AddIndex(
+            model_name='iplog',
+            index=olympia.amo.models.LongNameIndex(fields=['_ip_address'], name='log_activity_ip_ip_address_ba36172a'),
+        ),
+    ]
```

Comment:
The `RemoveIndex` followed by `AddIndex` is annoying, it's because the field was renamed but it's pointless to run for the database. Since the choices changes is also irrelevant (but harmless) for the database, maybe instead we should have:

```suggestion
    state_operations = [
        migrations.RemoveIndex(
            model_name='iplog',
            name='log_activity_ip_ip_address_ba36172a',
        ),
        migrations.AlterField(
            model_name='activitylog',
            name='action',
            field=models.SmallIntegerField(choices=[(1, 1), (2, 2), (3, 3), (4, 4), (5, 5), (6, 6), (7, 7), (8, 8), (9, 9), (12, 12), (16, 16), (17, 17), (18, 18), (19, 19), (20, 20), (21, 21), (22, 22), (23, 23), (24, 24), (25, 25), (26, 26), (27, 27), (28, 28), (29, 29), (31, 31), (32, 32), (33, 33), (34, 34), (35, 35), (36, 36), (37, 37), (38, 38), (39, 39), (40, 40), (41, 41), (42, 42), (43, 43), (44, 44), (45, 45), (46, 46), (47, 47), (48, 48), (49, 49), (53, 53), (60, 60), (61, 61), (62, 62), (98, 98), (99, 99), (100, 100), (101, 101), (102, 102), (103, 103), (104, 104), (105, 105), (106, 106), (107, 107), (108, 108), (109, 109), (110, 110), (120, 120), (121, 121), (128, 128), (130, 130), (131, 131), (132, 132), (133, 133), (134, 134), (135, 135), (136, 136), (137, 137), (138, 138), (139, 139), (140, 140), (141, 141), (142, 142), (143, 143), (144, 144), (145, 145), (146, 146), (147, 147), (148, 148), (149, 149), (150, 150), (151, 151), (152, 152), (153, 153), (154, 154), (155, 155), (156, 156), (157, 157), (158, 158), (159, 159), (160, 160), (161, 161), (162, 162), (163, 163), (164, 164), (165, 165), (166, 166), (167, 167), (168, 168), (169, 169), (170, 170), (171, 171), (172, 172)]),
        ),
        migrations.AddIndex(
            model_name='iplog',
            index=olympia.amo.models.LongNameIndex(fields=['_ip_address'], name='log_activity_ip_ip_address_ba36172a'),
        ),
    ]
    operations = migrations.SeparateDatabaseAndState(database_operations=[], state_operations=state_operations)

```